### DISCUSSION TITLE
Vendor Central Integration

### DISCUSSION THREAD

**Discussion by KryeKuzhinieri** [saleweaver/python-amazon-sp-api#858] [Q&A]
Hi everyone!

We have been using this package to fetch data from the Amazon Selling Partner API for a while and it has been working well. However,  we are dealing with an issue to integrate Vendor Central with the current implementation. The Vendor Central documentation states that the urls for this particular feature are different. So, we were wondering if there is a way to integrate them with the current python package? For example, in order to get the vendor central reports, in theory we could just change the base_urls and it should work or it requires an entire reimplementation of the package? 

Any help would be appreciated

https://developer-docs.amazon.com/sp-api/docs/vendor-central-urls

**Comment by saleweaver** [comment]
Just use the vendor clients in the package.

**Reply by KryeKuzhinieri** [reply]
I haven't seen anywhere any documentation on how to fetch reports for vendors clients. Could you point me out to the documentation or how to go about implementing this?
